site/content/docs/burp-to-zap-feature-map.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Burp to ZAP Feature Map
+type: page
+---
+
+Burp Suite is a popular commercial web app pentesting tool.
+It provides a free (closed source) Community edition and a paid for Professional edition.
+Many people are unaware that ZAP provides most of the features available in both the Community and Professional editions of Burp.
```

Comment:
```suggestion
Many people are unaware that ZAP provides most of the features available in both the Professional and Community editions of Burp.
```

Swap that because we rock!!!! :grinning:

Reply:
Agreed! And done 👍